resdk/resources/sample.py

```diff
@@ -225,3 +225,22 @@ def is_background(self):
             self._is_background = len(background_relations) > 0  # pylint: disable=len-as-condition
 
         return self._is_background
+
+    def duplicate(self, inherit_collection=False):
+        """Duplicate (make copy of) ``sample`` object.
+
+        :param inherit_collection: If ``True`` then duplicated
+            samples will be added to collections the original sample
```

Comment:
Since samples and collection now have m2o instead of m2m relation, this could be simplified. Something like: "If `True` then duplicated samples (and their data) will be added to collection of the original sample."